### ISSUE TITLE
Support to rails 7.1.0.alpha

### ISSUE THREAD

**Issue by vpereira** [flyerhzm/bullet#656]
Running my test suite locally with the upcoming rails version using [Next Rails] https://github.com/fastruby/next_rails, I'm getting the following error:

```
# RuntimeError:
#   Bullet does not support active_record 7.1.0.alpha yet
#   /home/foobar/.bundle/ruby/3.2.0/gems/bullet-7.0.7/lib/bullet/dependency.rb:35:in `active_record_version'
No examples found.
```

There are already plans to add support to, rails 7.1.0?

**Comment by flyerhzm** [comment]
@vpereira I usually support the rails versions only when it's public released. It may change the implementations in alpha and beta which bullet relies on.

**Comment by joshRpowell** [comment]
see #675

**Comment by flyerhzm** [comment]
@vpereira I just released bullet gem 7.1.0, which supports rails 7.1.0.
